Hotfix/panic button

MulvadT opened the pull request.

Commit:
feat: panic button

rasmim commented:
Hi @MulvadT, please rotate the secret when you make the changes. Also, add the new format for the config, without the actual secret.

Commit:
fix: secret in defatt config and parsed to frontend

Commit:
Updated to use redirect and flash message

This update is changed to use redirects instead of js function, this makes the webpage refreah instead, but are the same as the rest of the code.
Also the http codes should be looked at, but redirect only seems to work for 300 codes.
